Classes/Private/Manager/OCTManager.m

```diff
@@ -93,10 +88,34 @@ - (instancetype)initWithConfiguration:(OCTManagerConfiguration *)configuration
     _bootstrap = [self createSubmanagerWithClass:[OCTSubmanagerBootstrap class]];
     _chats = [self createSubmanagerWithClass:[OCTSubmanagerChats class]];
     _files = [self createSubmanagerWithClass:[OCTSubmanagerFiles class]];
+    _files.queue = _tox.queue;
     _friends = [self createSubmanagerWithClass:[OCTSubmanagerFriends class]];
     _objects = [self createSubmanagerWithClass:[OCTSubmanagerObjects class]];
     _user = [self createSubmanagerWithClass:[OCTSubmanagerUser class]];
 
+    OCTSubmanagerFriends *friends = [OCTSubmanagerFriends new];
+    friends.dataSource = self;
+    [friends configure];
+    _friends = friends;
+
+    OCTSubmanagerChats *chats = [OCTSubmanagerChats new];
+    chats.dataSource = self;
+    _chats = chats;
+
+    OCTSubmanagerFiles *files = [OCTSubmanagerFiles new];
+    files.dataSource = self;
+    [files configure];
+    files.queue = self.tox.queue;
+    _files = files;
+
+    OCTSubmanagerAvatars *avatars = [OCTSubmanagerAvatars new];
+    avatars.dataSource = self;
+    _avatars = avatars;
+
+    OCTSubmanagerObjects *objects = [OCTSubmanagerObjects new];
+    objects.dataSource = self;
+    _objects = objects;
+
```

Comment:
Please remove this block of code, submanagers are being created above with one-line method.